streams/us_stirr.m3u

```diff
@@ -19,239 +31,123 @@ http://dai.google.com/linear/hls/event/rtcMlf4RTvOEkaudeany5w/master.m3u8
 https://dai.google.com/linear/hls/event/Xu-ITJ2GTNGaxGn893mmWg/master.m3u8
 #EXTINF:-1 tvg-id="CBSNewsPittsburgh.us",CBSN Pittsburgh PA (720p)
 https://dai.google.com/linear/hls/event/i5SXVKI4QIuV-eF2XAH4FQ/master.m3u8
+#EXTINF:-1 tvg-id="",Channel Fight (1080p)
+https://d15wqvt0xm15k4.cloudfront.net/v1/master/9d062541f2ff39b5c0f48b743c6411d25f62fc25/ChannelFight-Nuestra/266.m3u8
```

Comment:
https://d15wqvt0xm15k4.cloudfront.net/scheduler/scheduleMaster/266.m3u8

That's a whole bunch of MuxIP streamers you've stumbled upon.

Reply:
Yeah, I was debating whether or not to use the "scheduleMaster" links, as many of them are for other services like Distro, but later decided to just keep them specific to Stirr for now.